content/v2.0/influxdb-templates/stacks/_index.md

```diff
@@ -39,52 +37,118 @@ Use stacks to save time in the following cases:
-4. Start an instance of InfluxDB, and then in your IDE, run `./install.sh`.
-   The install script finds your instance (`CONFIG_PROFILE.url`) and sets one up if the instance doesn't exist,
-   and then creates the stacks with the specified resources. This script creates separate stacks for buckets, dashboards, tasks, and Telegraf configurations;
-   however, you can create a stack with any combination of resources.
-   A message confirms your stacks were set up successfully and provides a link to your instance, for example `http://localhost:9999`.
-
-5. Click the link to open your InfluxDB instance and sign in. Your stacks should be applied to your instance.
-6. To start ingesting data into your instance, in your IDE, run `./start_telegraf.sh` script.
-   The script runs your Telegraf configurations and connects to your instance. You should see data coming into your instance.
-7. Open the `.github/workflows` directory, and make changes to `main.yml` as needed. This file uses a [GitHub Action](https://github.com/features/actions) to pull in some secrets
-   set in the GitHub project, grab the latest nightly version of InfluxDB, unzip the build, go to the `GITHUB_WORKSPACE`, and then
-   deploy the changes to your instance. Note the final line in this script `./install.sh prod` passes in a the `prod` configuration variable,
-   which grabs the INFLUX_TOKEN, INFLUX_ORG, and INFLUX_URL to install everything into the Cloud instance.
-8. Click **Actions** in the repository to see all the tasks in the `deploy-to-cloud` workflow.
-
-   Optionally, you can set up another continuous integration platform, such as [Circle CI](https://circleci.com/).
-
-#### Make a change in your repository
-
-Once you've set up your GitHub repository to back your InfluxDB instance, follow the steps below
-to make a change to your new repository and automatically deploy the change to your InfluxDB instances.
-
-1. In your integrated development environment (IDE), open the repository, make a change locally, and then test against your InfluxDB instance.
-2. Commit your change. For more information, see [GitHub docs about commits](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/committing-and-reviewing-changes-to-your-project#about-commits).
-3. Create a pull request. For more information, see [GitHub docs for creating a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
-4. Merge your pull request. For more information, see [GitHub docs for merging a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request). Your change is automatically deployed to your InfluxDB instance.
+As long as your deployment automation software allows you to run a custom script, you can set it up to run the same script you've built locally, but against another environment. Here is an example of using a custom Github Action to do this:
```

Comment:
...you can set a custom script? to run your local script against another environment.